java/query/src/test/java/sleeper/query/executor/QueryExecutorTest.java

```diff
@@ -743,114 +734,39 @@ public void shouldReturnCorrectDataWhenRecordsInMultipleFilesInMultiplePartition
         //           |           |           |
         //        "" +-----------+-----------+
         //           ""         "I"          null      (Dimension 1)
-        StateStore stateStore = getStateStore(schema);
-        Partition rootPartition = stateStore.getAllPartitions().get(0);
         // Add 4 records - record i is in the center of partition i
         Record record1 = createRecordMultidimensionalKey("D", "J", 10L, 100L);
         Record record2 = createRecordMultidimensionalKey("K", "H", 1000L, 10000L);
         Record record3 = createRecordMultidimensionalKey("C", "X", 100000L, 1000000L);
         Record record4 = createRecordMultidimensionalKey("P", "Z", 10000000L, 100000000L);
         List<Record> records = Arrays.asList(record1, record2, record3, record4);
-        ingestData(instanceProperties, stateStore, tableProperties, records.iterator());
         // Split the root partition into 2: 1 and 3, and 2 and 4
```

Comment:
This comment has been moved out of order, it should be next to where the partitions are split. The other comments that talk about how the partitions have been split have been deleted, so maybe this one should be too.

There's also an extra empty line below here.